packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -98,22 +98,25 @@ export class RetriableDocumentStorageService implements IDocumentStorageService,
         );
     }
 
-    private checkStorageDisposed() {
-        if (this._disposed) {
-            // pre-0.58 error message: storageServiceDisposedCannotRetry
-            throw new GenericError("Storage Service is disposed. Cannot retry", { canRetry: false });
-        }
-        return undefined;
-    }
-
     private async runWithRetry<T>(api: () => Promise<T>, callName: string): Promise<T> {
-        return runWithRetry(
+        const runResult = await runWithRetry2(
             api,
```

Comment:
we don't need a second abort mechanism here, but create an abort signal, and wire it to abort on dispose. that could happen in the onRetry handler if we do not have a better place